content/en/news/releases/1.21.x/announcing-1.21/change-notes/index.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Announcing Istio x.y.z
+linktitle: 1.21.0
+subtitle: Patch Release
+description: Istio x.y.z patch release.
+publishdate: 2020-07-29
+release: 1.21.0
+aliases:
+    - /news/announcing-1.21.0
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
```

Comment:
```suggestion
```

Reply:
what's the process to remove these warnings?

Reply:
It's part of the proof reading RMs do between running the tool and pasting them into the release notes. The release notes need spelling/grammatical fixes more often than not, the warning is mostly for you to know that it's a rough draft and needs reviewing and correcting before publishing.